org.civicrm.reqangular/src/common/decorators/ui-bootstrap/uib-modal.decorator.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env amd */
+
+define(function () {
+  'use strict';
+
+  return ['$delegate', '$document', function ($delegate, $document) {
+    var openFunction = $delegate.open;
```

Comment:
Name it something like `originalOpenFunction`, so that it is distinguishable from `open`.